### PR TITLE
added: copyright years to version.txt

### PR DESCRIPTION
**Pull request by notspiff**
Alternative to https://github.com/xbmc/xbmc/pull/14622

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
Can also be added to the NSIS installer. thx

**Comment by notspiff** [comment]
i don't see any entry for the years in the manifest and/or nsi file

**Comment by MartijnKaijser** [comment]
My mistake. Mixed something up

**Comment by Rechi** [comment]
jenkins build this please

File: xbmc/AppParamParser.cpp [hunk 1]
```diff
@@ -40,7 +41,7 @@ void CAppParamParser::Parse(const char* const* argv, int nArgs)
 void CAppParamParser::DisplayVersion()
 {
   printf("%s Media Center %s\n", CSysInfo::GetVersion().c_str(), CSysInfo::GetAppName().c_str());
-  printf("Copyright (C) 2005-2013 Team %s - http://kodi.tv\n", CSysInfo::GetAppName().c_str());
+  printf("Copyright (C) %s Team %s - http://kodi.tv\n", CSysInfo::GetAppName().c_str(), CCompileInfo::GetCopyrightYears());
```
Comment (hudokkow) [inline xbmc/AppParamParser.cpp]: Values are now swapped. Should be the other way around

Comment (notspiff) [reply xbmc/AppParamParser.cpp]: that was stupid..

**Pull request merged**